qml/harbour-hydrogen.qml

```diff
@@ -11,28 +11,36 @@ import Sailfish.WebView 1.0
 import Sailfish.WebEngine 1.0
 import Nemo.DBus 2.0
 import Nemo.Configuration 1.0
+import Nemo.Notifications 1.0
 import io.thp.pyotherside 1.5
 import "cover"
-import "components"
 
 ApplicationWindow {
     id: app
     cover: Component { HydrogenCover {} }
 
     allowedOrientations: Orientation.All
-    property int notificationCount: 0
-    property int oldUnreadCount: 0
     property var openingArgument
     property bool isSettingsAvailable: true
+
+    property int notificationCount: 0
+    property int oldUnreadCount: 0
     onNotificationCountChanged: {
-        if (oldUnreadCount < notificationCount && Qt.application.state !== Qt.ApplicationActive) {
-            notifier.quickNumberedNotification( "New Messages", notificationCount )
+        if (oldUnreadCount < notificationCount && appConfig.showNotifications) {
+            var tryToBeSilent = Qt.application.state == Qt.ApplicationActive;
+            var message = notificationComponent.createObject(null, {
+                'previewSummary': tryToBeSilent ? null: qsTr("New hydrogen message."),
+                'isTransient': !appConfig.stickyNotifications,
```

Comment:
That is a very good point - I just re-read the descripton from UI for "Sticky notifications".

In my opinion, since we don't (yet) deliver a notification per room, having them sticky by default is useful at the moment.
With that in mind, I thought that making them transient would be the other option.,

Now: what do you think - should I re-phrase the UI to my changes (e.g "Transient notifications", or actually make `lastNotificationId` play along only when sticky (and make that the default)? Third option: just remove UI setting and make all sticky for now. Thanks!

Reply:
Went with re-purposing transient vs sticky in settings